.gitignore

```diff
@@ -53,4 +53,4 @@ gradle-app.setting
 
 #Examples
 examples/**/Dependencies.toml
-examples/**/Config.toml
+#examples/**/Config.toml
```

Comment:
Why is this commented?

Reply:
I have to add `Config.toml` into the example. Hence, I enabled it.

Reply:
That can be enabled by adding the specific file. By disabling this, this can lead to people committing Config.toml with credentials. Lets revert it.